libs/domain/cart/src/models/cart.api.model.ts

```diff
@@ -21,6 +21,7 @@ export module ApiCartModel {
     thresholds?: unknown[];
     currency?: string;
     store?: string;
+    etag?: string;
```

Comment:
I'm thinking about more generic name in our UI model for cart, maybe `version` ?